proposals/lip-0054.md

```diff
@@ -783,19 +760,19 @@ def verify(trs: Transaction) -> None:
     if trsParams.chainID == getMainchainID() or trsParams.chainID == ownChainAccount.chainID:
         raise Exception("Chain ID is not valid.")
 
-    # The commands fails if the chain is not registered.
+    # The command fails if the chain is not registered.
     if chainAccount(trsParams.chainID) does not exist:
         raise Exception("Chain is not registered.")
 
-    # The commands fails if the chain is not terminated.
+    # The command fails if the chain is not terminated.
     if terminatedStateAccount(trsParams.chainID) does not exist or terminatedStateAccount(trsParams.chainID).initialized == False:
```

Comment:
I think the second part of this statement `or terminatedStateAccount(trsParams.chainID).initialized == False` can not actually happen: For the mainchain, terminated state accounts are always initialized (if they exist..); since we define this command only for mainchain , I think it is redundant and can be removed.

Note that even if we allow direct channels in the future, then again, `createTerminatedStateAccount` will always initialize the terminated account (because a chainAccount for the terminated chain would exist).

It is fine for me to keep it for extra security, but if I am not missing something, it can be removed.

Reply:
I think you are right, I removed the extra check